plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/NugetGalleryFacetImpl.java

```diff
@@ -244,7 +246,7 @@ protected void doPutContent(final String id, final String version, final Supplie
         component != null && tx.browseAssets(component).iterator().hasNext(),
         "Component metadata does not exist yet"
     );
-    createOrUpdateAssetAndContents(tx, component, streamSupplier.get(), null);
+    createOrUpdateAssetAndContents(tx, component, streamSupplier.get(), null, content.getAttributes());
```

Comment:
if all we need here are the attributes, maybe change signature to take in just that?